src/GeositeFramework/Models/Geosite.cs

```diff
@@ -152,7 +153,14 @@ private static List<JsonData> GetPluginConfigurationData(IEnumerable<string> plu
             if (jsonObj["headerLinks"] != null)
             {
                 HeaderLinks = jsonObj["headerLinks"]
-                    .Select(ExtractLinkFromJson).ToList();
+                    .Where(element => element["text"].Value<String>() != "GO TO")
```

Comment:
This seems a little too lose, I think that we should update the region schema to allow for an explicit type of `regionList` and not infer it from the label text.